lib/ecl/ecl_file_kw.c

```diff
@@ -323,4 +336,88 @@ void ecl_file_kw_inplace_fwrite( ecl_file_kw_type * file_kw , fortio_type * fort
 }
 
 
+void ecl_file_kw_fwrite( const ecl_file_kw_type * file_kw , FILE * stream ) {
+  int header_length = strlen( file_kw->header );
+  for (int i=0; i < ECL_STRING8_LENGTH; i++) {
+    if (i < header_length)
+      fputc( file_kw->header[i], stream );
+    else
+      fputc( ' ' , stream );
+  }
```

Comment:
The header will have `len(header) <= ECL_STRING8_LENGTH`. For this particular we pad with spaces if the header is shorter - to guarantee that the on-disk size of one `ecl_file_kw_type` instance will have a deterministic size - that should lead to better performance when many (~ 100000) instances should be loaded.